### PR TITLE
pathogen dots

### PR DESCRIPTION
**Pull request by cdebourcy**
https://chanzuckerberg.invisionapp.com/share/WRO751DA2QT#/326240968_2-1_Sample_Report_Copy_4
oct-21-2018 15-41-41

### CODE REVIEW

File: app/assets/src/components/views/report/PathogenPreview.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <Label circular color={CATEGORIES[type]["color"]} key={type} />
+          );
+        })}
+        <span className="pathogen-count">{totalCount}</span>
+      </span>
+    );
+    return (
+      <BasicPopup
+        trigger={display}
+        content="contains flagged pathogen species"
```
Comment (jshoe) [inline app/assets/src/components/views/report/PathogenPreview.jsx]: nit: could you capitalize 'Contains' for consistency with other popups

Comment (cdebourcy) [reply app/assets/src/components/views/report/PathogenPreview.jsx]: done

**Comment by jshoe** [comment]
- could you align the dots a little lower with some margin-top? And make the count number a little smaller so it sticks out less?
- also could you remove the yarn.lock diff (or re-checkout that file from master)? you didn't actually install anything right? I'm going to try to lock down the integrity hash thing since Tiago did a recent commit to keep them added

**Comment by jshoe** [comment]
or maybe yarn.lock will go away if you merge/rebase

File: app/assets/src/components/views/report/PathogenPreview.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import BasicPopup from "../../BasicPopup";
+import Label from "../../ui/labels/Label";
+import { CATEGORIES } from "../../ui/labels/PathogenLabel";
+
+const PathogenPreview = ({ tag2count }) => {
```
Comment (tfrcarvalho) [inline app/assets/src/components/views/report/PathogenPreview.jsx]: tag2Count :)

**Comment by tfrcarvalho** [comment]
@cdebourcy Can you reduce the size of the dots to match Jenn's specs better. We actually start adding her to these reviews.

File: app/assets/src/components/views/report/PathogenPreview.jsx (hunk 2 above)
Comment (cdebourcy) [reply app/assets/src/components/views/report/PathogenPreview.jsx]: ok changed

**Comment by cdebourcy** [comment]
> or maybe yarn.lock will go away if you merge/rebase

OK fixed thanks

**Comment by jshoe** [comment]
@tfrcarvalho you mean you personally don't like the number outside?

**Comment by tfrcarvalho** [comment]
@jshoe Yeah, thanks for correcting me! 
In the meantime, chatted with Jenn and she is afraid will be two bulky and/or busy. I am fine moving on with the number outside as soon as the small dot alignment/size tweaks are done.

**Comment by cdebourcy** [comment]
@tfrcarvalho Great, I've made those tweaks!

**Pull request merged**